static/js/chart/draw_map_utils.ts

```diff
@@ -425,3 +504,64 @@ export function highlightPlaceToggle(
     container.classed(HOVER_HIGHLIGHTED_CLASS_NAME, shouldHighlight);
   }
 }
+
+/**
+ * Create function that generates tooltip content
+ * @param chartData geojson and observation data for all layers of the map
+ */
+export function getTooltipHtmlFn(
+  chartData: MapChartData
+): (place: NamedPlace) => string {
+  // build mapping of all values, metadata, and variable units
+  // for a place/variable, to show in tooltip
+  const allDataValues: { [placeDcid: string]: { [variable: string]: number } } =
+    {};
+  const allMetadataValues: {
+    [placeDcid: string]: { [variable: string]: DataPointMetadata };
+  } = {};
+  const units: { [variable: string]: string } = {};
```

Comment:
Good catch! This should get populated, added the logic to do that.

Reply:
nit: I don't think you need this though, chartData.units is a map of sv dcid to unit?

Reply:
`chartData` doesn't have a units field actually, but I think you're right it would make sense for sv->unit mapping to get stored there. Will hold that refactoring off for a followup PR, this one is large enough as it is.